README.md

```diff
@@ -1,50 +1,67 @@
-# jquery-inputcloak
+# jquery-inputCloak
 
 ## What is it?
 
 A simple plugin that cloaks/masks an input, similar to a password box.
 
-Ex. 123-45-6789 becomes xxx-xx-6789.
+e.g. 123-45-6789 becomes xxx-xx-6789.
 
 ## How do I use it?
 
-Include relevant js files - jQuery and jquery.inputcloak.min.js
+### Include js files:
+* jQuery
+* jquery-inputCloak.min.js
 
-Select page elements and apply cloaking to them, with default settings or customised ones, like
+### Select DOM elements and apply cloaking to them
+
+with default settings
 ```javascript
 $('.yourpasswordinput').inputCloak();
 ```
-OR
+or with tailored settings, like
 ```javascript
-$('.yourpasswordinput').inputCloak({settings});
+$('.yourpasswordinput').inputCloak(settings);
 ```
-where settings is an object with as many as relevant of the following members:
-\(default is shown first\)
-
- type: 'all' \('ssn','credit' or 'all'
-   for display respectively like \*\*\*-\*\*-1234,\*1234 or \*\*\*\*,
-   irrelevant if customCloak is used\)<br />
- symbol: '\*' \(any single-char e.g. '\*',\u2022 \(dot\), 'x'\)<br />
- delay: 0 \(mSec delay between blur event and cloak-application\)<br />
- revealOnFocus: true \(or false\)<br />
- cloakOnBlur: true \(or false\)<br />
- customCloak : undefined \(or name of callback, as
+where settings is an object with any of the following members:
+
+| Name | Options | Description |
+| ------- | ----- | ----------- |
+| type       | String 'ssn','credit','see4','see1' or 'all'      | Default: 'ssn'. for display respectively like \*\*\*-\*\*-1234, \*1234,\*\*\*\*1234, \*\*\*\*4 or \*\*\*\*, irrelevant if customCloak is used |
+| symbol     | any single-char e.g. '\*',\u2022 \(dot\), 'x'  | Default: '\*'. Type of cloak symbol |
+| delay      | Integer  | Default: 0. mSec delay between blur event and cloak-application |
+| revealOnFocus  | Boolean        | Default: true.  |
+| cloakOnBlur    | Boolean        | Default: true.  |
+| customCloak    | Function       | Default: undefined. Called from blur-event handler, with parameters value, $element, $cloakedElement.  |
+
+An example customCloak callback is
 ```javascript
-   callback(value, $element, $cloakedElement) {
-     determine cloakedValue from value
-     $cloakedElement.val(cloakedValue);
-     $element.val(value);
+   function mycloaker(value, $element, $cloakedElement) {
+     var customMaskValue = some_string_derived_from(value);
+     $cloakedElement.val(customMaskValue);
    }
 ```
-  \)
+### Need more examples?
+
+Head on over [here](http://ermish.github.io/jquery-inputcloak).
 
-Head on over to [github](http://ermish.github.io/jquery-inputcloak) for examples.
+## API
 
+Properties added to each selected element
+```
+.settings = object containing default parameters as modified by initiating application
+._defaults = object containing default parameters
+._name = constant name of this plugin
+```
+Methods added to each selected element
```

Comment:
Change to something like 

###Available Functions

### .cloak()

`js example`

details here.